addons/l10n_id/models/account_move.py

```diff
@@ -19,7 +19,9 @@ class AccountMove(models.Model):
         help="Transaction Number stored.",
         readonly=True,
         export_string_translation=False,
+        related="l10n_id_qris_transaction.qris_invoice_details"
```

Comment:
Note that this will require a migration script.
You will want to create the l10n_id.qris.transaction based on the content of the details (to be safe), then drop the details column.

Reply:
Also I think this field can be removed